<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPilotForm</class>
 <widget class="QFrame" name="CPilotForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>57</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pilot info</string>
  </property>
  <layout class="QGridLayout" name="gl_PilotInfo">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_Id">
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_Id"/>
   </item>
   <item row="0" column="2">
    <widget class="swift::gui::CTickLabel" name="lblp_Id">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="le_Password">
     <property name="maxLength">
      <number>100</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="swift::gui::CTickLabel" name="lblp_Password">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lbl_Password">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="swift::gui::CTickLabel" name="lblp_RealName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="lbl_HomeAirport">
     <property name="toolTip">
      <string>Home airport, e.g. &quot;KLAX&quot;</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_RealName">
     <property name="toolTip">
      <string>Real name</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_RealName">
     <property name="placeholderText">
      <string>e.g. Joe Doe</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="swift::gui::CTickLabel" name="lblp_HomeAirport">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="5" colspan="2">
    <widget class="swift::gui::components::CAirportSmallCompleter" name="comp_HomeAirport">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pb_UnhidePassword">
     <property name="toolTip">
      <string>unhide password</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../misc/misc.qrc">
       <normaloff>:/diagona/icons/diagona/icons/question-button.png</normaloff>:/diagona/icons/diagona/icons/question-button.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::CTickLabel</class>
   <extends>QLabel</extends>
   <header>gui/ticklabel.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CAirportSmallCompleter</class>
   <extends>QFrame</extends>
   <header>gui/components/airportsmallcompleter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Id</tabstop>
  <tabstop>le_Password</tabstop>
  <tabstop>pb_UnhidePassword</tabstop>
  <tabstop>le_RealName</tabstop>
  <tabstop>comp_HomeAirport</tabstop>
 </tabstops>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
